hack/aks/Makefile

```diff
@@ -272,6 +272,38 @@ dualstack-overlay-byocni-up: rg-up overlay-net-up ## Brings up an dualstack Over
 		--yes
 	@$(MAKE) set-kubeconf
 
+cilium-dualstack-up: rg-up overlay-net-up ## Brings up a Cilium Dualstack Overlay cluster with Linux node only
```

Comment:
Will there be another PR that also adds in the configmap for cilium dualstack, or are they the same?

Reply:
yes @rjdenney is working on an e2e pipeline stage for dualstack that will include a configmap with values for ipv6